README.md

```diff
@@ -0,0 +1,216 @@
+<a name="#readme-top"></a>
```

Comment:
- [ ] On your readme, these links won't work as they just lead to the repo's main page:

![image](https://github.com/diarisdiakite/my-booking-app/assets/57421823/fb044311-009f-4eff-9209-3303f64f3dac)

Kindly fix this links so they lead to what is specified in the labels. 👍